app/cafe.py

```diff
@@ -0,0 +1,22 @@
+import datetime
+from app.errors import (NotVaccinatedError,
+                        OutdatedVaccineError, NotWearingMaskError)
+
+
+class Cafe:
+    def __init__(self, name: str) -> None:
+        self.name = name
+
+    def visit_cafe(self, visitor: dict) -> None | str:
+        if "vaccine" not in visitor:
+            raise NotVaccinatedError(f"{visitor["name"]}"
+                                     f" is not vaccinated.")
+        expiration_date = visitor["vaccine"]["expiration_date"]
+        if expiration_date < datetime.date.today():
+            raise OutdatedVaccineError(f"{visitor["name"]}'s"
+                                       f" vaccine is outdated (expired"
+                                       f" on {expiration_date}).")

```

Comment:
There is a syntax error in the string formatting. You should use single quotes around the key `name` inside the f-string or escape the double quotes. For example: `f"{visitor['name']}'s vaccine is outdated (expired on {expiration_date})."`.